corehq/apps/cloudcare/templates/form_entry/templates.html

```diff
@@ -315,10 +315,15 @@ <h1 class="title" data-bind="text: title, visible: !showInFormNavigation()"></h1
   <!-- ko if: isLabel -->
   <div class="form-group">
     {# appearance attributes TEXT_ALIGN_CENTER TEXT_ALIGN_RIGHT #}
-    <div class="info col-sm-12" data-bind="css: {
```

Comment:
I am slightly confused how the selector above worked before. Shouldn't there have been a `for` attribute on this tag?

Reply:
I branched this off of https://github.com/dimagi/commcare-hq/pull/34289 accidentally, so you're seeing that bug fix again.

Reply:
I still don't understand how it selected it before the change though. I get why it does now.

Reply:
It didn't, that's the bug. It could select non-label questions since they use [this markup](https://github.com/dimagi/commcare-hq/blob/79284ff3f5f47d31c560723b51438fc8f5238b8c/corehq/apps/cloudcare/templates/form_entry/templates.html#L237-L242), with both a for attribute and an id. But label questions didn't have either of those.